docs/docs/snippets/providers/fn/lazy-inject-declaration.ts

```
@@ -0,0 +1,25 @@
+import {injectable} from "@tsed/di";
+
+export default class MyModule {
+  private initialized: boolean = false;
```

Biome

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)